frontend/src/pages/projects/notebook/ConnectedNotebookNames.tsx

```diff
@@ -32,13 +32,13 @@ const ConnectedNotebookNames: React.FC<ConnectedNotebookNamesProps> = ({
   }
 
   return (
-    <List isPlain>
+    <LabelGroup isCompact>
```

Comment:
I tested on the PF website, not sure if the doc is outdated, but I am pretty sure there is no need to add `isCompact` on `LabelGroup`. There is no hurt to add it though, just a small nit.